pkg/apps/alert/latest/label.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright (C) 2019 Synopsys, Inc.
+
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements. See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership. The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied. See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+package alert
+
+import "github.com/blackducksoftware/synopsys-operator/pkg/util"
+
+// GetLabel will return the label
+func (a *SpecConfig) getLabel(name string) map[string]string {
+	return map[string]string{
```

Comment:
In Black Duck, we used to have getVersionLabel and getLabel. Probably we should consider having the same for an Alert instance as well. Maybe a new PR...

Reply:
I've added a TODO in the label file.  I'll address in another PR for 6.1 or 9.0